frontend/package.json

```diff
@@ -95,6 +95,7 @@
     "uuid": "8.3.2",
     "validator": "13.7.0",
     "whatwg-fetch": "3.6.2",
+    "xlsx": "https://cdn.sheetjs.com/xlsx-0.20.2/xlsx-0.20.2.tgz",
```

Comment:
I do not think this will work in our downstream builds because Cachito probably does not cache this package. They have not really provided a good explanation of why their package is not being published on npm.
https://github.com/SheetJS/sheetjs/issues/2667

This package does a lot more than we need, and while I'm usually in favour of using a library, since we already have CSV export functionality in our UI that doesn't depend on a package, I think we should share that code instead.